core/common/src/main/java/alluxio/Constants.java

```diff
@@ -118,7 +118,8 @@
   public static final String KEY_VALUE_WORKER_CLIENT_SERVICE_NAME = "KeyValueWorkerClient";
 
   public static final String REST_API_PREFIX = "v1/api";
-  public static final String LOGGER_TYPE = PropertyKey.LOGGER_TYPE;
+  public static final String LOGGER_TYPE = PropertyKey.LOGGER_TYPE.toString();
```

Comment:
otherwise any file getting a logger will have 
```java
protected static final Logger LOG = LoggerFactory.getLogger(PropertyKey.LOGGER_TYPE.toString());
```
which looks verbose to me.